src/main/java/com/piekoszek/nowaksharedrent/jwt/JwtServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.piekoszek.nowaksharedrent.jwt;
+
+import com.piekoszek.nowaksharedrent.jwt.exceptions.InvalidTokenException;
+import com.piekoszek.nowaksharedrent.time.TimeService;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.MalformedJwtException;
+import io.jsonwebtoken.security.SignatureException;
+import lombok.Setter;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+
+@Setter
+
```

Comment:
Redundant empty line (it doesn't bother me 2 much, but I know a lot of ppl who are crazy about formatting...)